fpdf/fonts.py

```diff
@@ -136,7 +136,15 @@ def __init__(
             fill_color,
         )
         self.t_margin = t_margin or 0
-        self.l_margin = l_margin or 0
+
+        # added support for 'Align' and 'str' type values for l_margin
+        if isinstance(l_margin, (Align, int)):
+            self.l_margin = l_margin
+        elif isinstance(l_margin, str):
+            self.l_margin = Align.coerce(l_margin)
+        else:
+            self.l_margin = 0
```

Comment:
```suggestion
        self.l_margin = Align.coerce(l_margin)
```
Isn't this one line enough?

Reply:
Hi @Lucas-C,

I dont think Align.coerce can handle integer values, so I added conditions for the 3 possible types integer, string and Align.